ReverseTicTacToe/UI/ConsoleUI.cs

```diff
@@ -33,17 +29,145 @@ public  void Init()
         }
         public  void Run()
         {
+            
             WriteLine("Welcome to Reverse Tic Tac Toe!" + Environment.NewLine);
-            getBoardSizeFromUser();
-            getGameTypeFromUser();
-            m_Game = new Game(m_BoardSize, m_GameType);
-            while(true)
+            int BoardSize =  getBoardSizeFromUser();
+            Game.eGameTypes GameType = getGameTypeFromUser();
+            m_Game = new Game(BoardSize, GameType);
+           // while(true)
+            //{
+            printGameBoard();
+            playTurn();
+            ReadLine();
+            
+            //}
+        }
+
+        private void playTurn()
+        {
+            int row = 0;
+            int col = 0;
+            //int return status: continue, place_taken, finished game
+            string turnMsg = GetTurnMessage();
+            
+            if(m_Game.CurrentTurn == 1 || m_Game.GameType == Game.eGameTypes.PVP)
+            {
+                string userTurnInput;
+                do
+                {
+                    WriteLine(turnMsg);
+                    userTurnInput = ReadLine();
+                }
+                while(!turnInputIsValid(userTurnInput, out row, out col));
+            }
+            if (row == -1 && col == -1)
```

Comment:
chage to enum/consts?

Reply:
changed to const:
private const int QUIT = -1